src/addon/state.cpp

```diff
@@ -54,19 +52,16 @@ addon_tracking_info get_addon_tracking_info(const addon_info& addon)
 			t.installed_version = get_addon_version_info(id);
 			if(addon.versions.size() > 0) {
 				t.remote_version = *addon.versions.begin();
-			} else {
-				t.remote_version = version_info(0,0,0);
 			}
 		}
 
-		if(t.remote_version == t.installed_version) {
+		if(t.remote_version == version_info()) {
```

Comment:
On second thought since there is `addon.local_only` we can just use that...

Reply:
Updated.